pyproject.toml

```diff
@@ -32,3 +32,9 @@ target_version = [
     "py310",
     "py311",
 ]
+
+[tool.pytest.ini_options]
+addopts = "--verbose --color=yes --durations=10"
+asyncio_mode = "auto"
+# Ignore thousands of tests in dependencies installed in a virtual environment
+norecursedirs = "lib lib64"
```

Comment:
This is perhaps simplified as `testpaths = [ "oauthenticator/tests" ]` rather than _excluding_ some files